test/integration/reboot_test.lua

```diff
@@ -198,3 +198,15 @@ function g.test_invalid_config()
         " not in clusterwide config, no idea what to do now"
     })
 end
+
+function g.test_advertise_change()
+    local master = g.cluster.main_server
+    master:stop()
+
+    master.env['TARANTOOL_ADVERTISE_URI'] = '127.0.0.1:13310'
+    master.net_box_uri = '127.0.0.1:13310'
+    master:start()
+
+    helpers.wish_state(master, 'RolesConfigured')
```

Comment:
It would be great to specify timeout here

Reply:
Done